docs/process/prd.md

```diff
@@ -0,0 +1,226 @@
+
+Finally, ICU4X will offer a powerful and flexible data management system, enabling sophisticated software release models and long-lived software processes to maintain data selection and updates.
+
+## Stakeholders
+
+The initial stakeholders are Unicode, Google and Mozilla.
+
+Unicode provides the knowledge, experience and guidance, and a non-corporate plane on which such industry wide effort can be developed.
+Mozilla and Google provide engineering resources driving the development and offering business needs, product environment, and high-fidelity opportunities to validate the project against popular mature software.
+
+As the project matures, we hope to attract three more classes of stakeholders:
+
+### Internationalization Community
+
+Thanks to Unicode guidance, strong engineering resources, mature CLDR base, and leveraging the experience of ECMA-402, ICU4C and ICU4J, ICU4X has a chance to expose the latest, modern and well designed APIs based on the lessons-learned of the last 30 years of the industry development.
+
+ICU4X will enable high quality, rich and modern internationalization for environments and software deployment types which are unable to benefit from the currently existing industry options.
+
+With those options at hand, ICU4X should become an attractive project for the widely understood Internationalization Community, which currently often maintains pieces of ICU-like APIs for their target needs, or a patchwork of wrappers and heavily fine-tuned deployments of ICU.
+
+### Major Software Vendors
+
+Besides Google and Mozilla, many other organizations aim to target their products and services at the whole world.
+
+Currently, most of them consume ICU, or maintain their own equivalent of such for environments where monolithic C++ and Java are not providing the right payload or safety characteristics.
+
+With the maturing of ICU4X we hope to attract those organizations to consider ICU4X and contribute to it.
+
+### Rust Community
+
+Rust programming language has evolved one of the most unique and [highly productive software communities in the world](https://stackoverflow.blog/2020/01/20/what-is-rust-and-why-is-it-so-popular/). Rust has been voted the most loved programming language in an annual StackOverflow survey for [four years in a row](https://stackoverflow.blog/2020/01/20/what-is-rust-and-why-is-it-so-popular/).
+
+Internationalization is a notoriously challenging domain of software, and the Rust community has proven itself to handle challenging problems very well. Currently the ecosystem uses Unicode, and ICU-like APIs in many crucial places, including rustc compiler itself and multiple high quality layout implementations such as [xi-editor](https://github.com/xi-editor/xi-editor).
+With the rising focus on higher-level libraries such as [GUI toolkits](https://www.areweguiyet.com/), [game and web engine](https://arewegameyet.rs/) components, the need for high-quality internationalization solutions will become needed.
```

Comment:
> Suggestion

"The need for {x} will become needed" reads a bit redundantly. 

I might suggest something like, "high-quality internationalization solutions are increasingly in demand."